tensorrt_llm/_torch/models/modeling_llava_next.py

```diff
@@ -51,6 +51,195 @@ def __init__(self,
 
         self.image_token_index = model_config.image_token_index
         self.vocab_size = model_config.vocab_size
+        self.vision_feature_select_strategy = getattr(
+            model_config, "vision_feature_select_strategy",
+            "default")
+        self.config = model_config.vision_config
+
+    def _get_num_unpadded_features(
```

Comment:
Could we have unit test(s) for this function?

Reply:
Sure. Added.